required/latex-lab/latex-lab-math.dtx

```diff
@@ -247,8 +247,15 @@
 %  It can be suppressed by an option with
 %  \texttt{math/tex/AF=false}, see below. 
 %  
-%  The MathML is attached if a file |\jobname-mathml.html| is found and
-%  if it contains a suitable MathML snippet for the current formula. 
+%  The MathML is attached if the files |\jobname-mathml.html| and/or 
+%  |\jobname-luamml-mathml.html| are found 
+%  and if they contains a suitable MathML snippet for the current formula. 
+%  If the files contain more than one suitable snippet (as identified by the hash)
+%  the first one is used. 
```

Comment:
Fine for the present, but normally I'd expect to have overrides based on _later_ files, i.e. that the last version would win (so you read the auto-generated code first then move on to manual adjustments).

Reply:
the current code already uses the first version as it makes processing easier: We embed the first fragment for a hash and then mark the hash as used and can ignore all the following. If we would have to wait until the end, we would have to keep all fragments in memory.